samples/net/wifi/apsta_mode/src/main.c

```diff
@@ -118,21 +110,23 @@ static int enable_ap_mode(void)
 	}
 
 	LOG_INF("Turning on AP Mode");
-	ap_config.ssid = (const uint8_t *)WIFI_AP_SSID;
-	ap_config.ssid_length = strlen(WIFI_AP_SSID);
-	ap_config.psk = (const uint8_t *)WIFI_AP_PSK;
-	ap_config.psk_length = strlen(WIFI_AP_PSK);
+	ap_config.ssid = (const uint8_t *)CONFIG_WIFI_AP_SSID;
+	ap_config.ssid_length = strlen(CONFIG_WIFI_AP_SSID);
```

Comment:
Instead of `strlen` do `sizeof(CONFIG_....) - 1` so that the value is created at built time.